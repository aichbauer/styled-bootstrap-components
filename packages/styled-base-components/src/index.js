import { BootstrapBaseCss } from './components';

export { BootstrapBaseCss };
