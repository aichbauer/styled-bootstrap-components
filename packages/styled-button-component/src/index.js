import {
  Button,
  LinkButton,
} from './components/Button';
import { ButtonGroup } from './components/ButtonGroup';
import { ButtonToolbar } from './components/ButtonToolbar';

export {
  Button,
  LinkButton,
  ButtonGroup,
  ButtonToolbar,
};
