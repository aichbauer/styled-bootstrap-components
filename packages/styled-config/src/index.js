import { theme } from './theme';
import {
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
} from './utils';

export {
  theme,
  // utils
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
};
