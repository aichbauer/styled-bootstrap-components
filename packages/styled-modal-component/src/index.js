import { Modal } from './components/Modal';
import { ModalWrapper } from './components/ModalWrapper';
import { ModalBody } from './components/ModalBody';
import { ModalContent } from './components/ModalContent';
import { ModalDialog } from './components/ModalDialog';
import { ModalFooter } from './components/ModalFooter';
import { ModalHeader } from './components/ModalHeader';
import { ModalTitle } from './components/ModalTitle';
import { ModalBackdrop } from './components/ModalBackdrop';

export {
  Modal,
  ModalWrapper,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  ModalBackdrop,
};
