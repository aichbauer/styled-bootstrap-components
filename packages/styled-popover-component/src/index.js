import { PopoverWithoutTransition, Popover } from './components/Popover';
import { PopoverArrow } from './components/PopoverArrow';
import { PopoverBody } from './components/PopoverBody';
import { PopoverHeader } from './components/PopoverHeader';

export {
  PopoverWithoutTransition,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
};
