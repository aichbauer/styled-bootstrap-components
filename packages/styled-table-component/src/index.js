import { Table } from './components/Table';
import { Tr } from './components/Tr';

export {
  Table,
  Tr,
};
