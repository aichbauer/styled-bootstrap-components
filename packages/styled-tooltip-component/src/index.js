import { TooltipWithoutTransition, Tooltip } from './components/Tooltip';
import { TooltipArrow } from './components/TooltipArrow';
import { TooltipInner } from './components/TooltipInner';

export {
  TooltipWithoutTransition,
  Tooltip,
  TooltipArrow,
  TooltipInner,
};
