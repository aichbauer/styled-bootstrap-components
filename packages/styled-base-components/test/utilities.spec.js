import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Div } from '../src';

test('Styles match Div', () => {
  const div = renderer.create(<Div />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with colored text and background', () => {
  const div = renderer.create(<Div textLight bgDark />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with padding and margin', () => {
  const div = renderer.create(<Div m1 p1 />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with margin and size', () => {
  const div = renderer.create(<Div m0 mMd2 />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with auto margin', () => {
  const div = renderer.create(<Div mlAuto />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with paddings', () => {
  let div;

  div = renderer.create(<Div p1 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div pl1 pr1 pt1 pb1 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div px1 py1 />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with rounded', () => {
  let div;

  div = renderer.create(<Div rounded />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div roundedLeft roundedRight />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div roundedTop roundedBottom />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with roundedCircle', () => {
  const div = renderer.create(<Div roundedCircle />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with roundedPill', () => {
  const div = renderer.create(<Div roundedPill />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with colored border', () => {
  const div = renderer.create(<Div border borderPrimary />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with 0 border', () => {
  const div = renderer.create(<Div border0 />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with hidden text', () => {
  const div = renderer.create(<Div textHide>text</Div>).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with truncated text', () => {
  const div = renderer.create(<Div textTruncate>text</Div>).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with word break', () => {
  const div = renderer.create(<Div textWordBreak>text</Div>).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with monospace', () => {
  const div = renderer.create(<Div textMonospace />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with italic', () => {
  const div = renderer.create(<Div textItalic />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with clearfix', () => {
  const div = renderer.create(<Div clearfix />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with text decoration', () => {
  const div = renderer.create(<Div textDecorationNone />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with text reset', () => {
  const div = renderer.create(<Div textReset />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with float', () => {
  let div;

  div = renderer.create(<Div floatRight />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div floatLeft />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div floatNone />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with transform', () => {
  let div;

  div = renderer.create(<Div textLowercase />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textUppercase />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textCapitalize />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with text wrap', () => {
  let div;

  div = renderer.create(<Div textWrap />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textNoWrap />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with text align', () => {
  let div;

  div = renderer.create(<Div textCenter />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textLeft />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textRight />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textJustify />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with vertical align', () => {
  let div;

  div = renderer.create(<Div textBaseline />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textTop />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textBottom />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textTextTop />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textTextBottom />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with width/height', () => {
  let div;

  div = renderer.create(<Div w100 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div h50 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div mw75 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div mh25 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div wAuto />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with overflow', () => {
  let div;

  div = renderer.create(<Div overflowAuto />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div overflowHidden />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with text weight', () => {
  let div;

  div = renderer.create(<Div textWeightBold />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textWeightBolder />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textWeightNormal />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textWeightLight />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div textWeightLighter />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with position', () => {
  let div;

  div = renderer.create(<Div positionStatic />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div positionRelative />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div positionAbsolute />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div positionFixed />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div positionSticky />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with shadow', () => {
  let div;

  div = renderer.create(<Div shadowNone />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div shadowSmall />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div shadow />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div shadowLarge />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with order', () => {
  const div = renderer.create(<Div order1 />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with flex direction', () => {
  let div;

  div = renderer.create(<Div flexRow />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexRowReverse />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexColumn />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexColumnReverse />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with justify content', () => {
  let div;

  div = renderer.create(<Div justifyContentStart />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div justifyContentEnd />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div justifyContentCenter />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div justifyContentSpaceBetween />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div justifyContentSpaceAround />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div justifyContentSpaceEvenly />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with align', () => {
  ['Start', 'End', 'Center', 'Baseline', 'Stretch'].forEach((v) => {
    const props = {
      [`alignSelf${v}`]: true,
      [`alignItems${v}`]: true,
      [`alignContent${v}`]: true,
    };

    const div = renderer.create(<Div {...props} />).toJSON();
    expect(div).toMatchSnapshot();
  });
});

test('Styles match Div with flex grow', () => {
  let div;

  div = renderer.create(<Div flexGrow0 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexGrow />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with flex shrink', () => {
  let div;

  div = renderer.create(<Div flexShrink0 />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexShrink />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with flex wrap', () => {
  let div;

  div = renderer.create(<Div flexWrap />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexNoWrap />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div flexWrapReverse />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with visibilities', () => {
  let div;

  div = renderer.create(<Div visible />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div invisible />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Div with resize', () => {
  let div;

  div = renderer.create(<Div resizeNone />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div resizeBoth />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div resizeHorizontal />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div resizeVertical />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match Divs with displays', () => {
  let div;

  div = renderer.create(<Div dNone />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dInline />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dInlineBlock />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dBlock />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dTable />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dTableCell />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dTableRow />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dFlex />).toJSON();
  expect(div).toMatchSnapshot();

  div = renderer.create(<Div dInlineFlex />).toJSON();
  expect(div).toMatchSnapshot();
});

test('Styles match hidden Div', () => {
  const div = renderer.create(<Div hidden />).toJSON();
  expect(div).toMatchSnapshot();
});
