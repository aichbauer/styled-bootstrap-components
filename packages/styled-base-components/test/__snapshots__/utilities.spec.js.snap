// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles match Div 1`] = `
<div
  className=""
/>
`;

exports[`Styles match Div with 0 border 1`] = `
.c0 {
  border-left: 0;
  border-right: 0;
  border-top: 0;
  border-bottom: 0;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with align 1`] = `
.c0 {
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  -webkit-align-self: start;
  -ms-flex-item-align: start;
  align-self: start;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with align 2`] = `
.c0 {
  -webkit-align-content: end;
  -ms-flex-line-pack: end;
  align-content: end;
  -webkit-align-items: end;
  -webkit-box-align: end;
  -ms-flex-align: end;
  align-items: end;
  -webkit-align-self: end;
  -ms-flex-item-align: end;
  align-self: end;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with align 3`] = `
.c0 {
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with align 4`] = `
.c0 {
  -webkit-align-content: baseline;
  -ms-flex-line-pack: baseline;
  align-content: baseline;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  -webkit-align-self: baseline;
  -ms-flex-item-align: baseline;
  align-self: baseline;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with align 5`] = `
.c0 {
  -webkit-align-content: stretch;
  -ms-flex-line-pack: stretch;
  align-content: stretch;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with auto margin 1`] = `
.c0 {
  margin-left: auto;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with clearfix 1`] = `
.c0 {
  display: block;
  content: "";
  clear: both;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with colored border 1`] = `
.c0 {
  border-left: 1px solid;
  border-right: 1px solid;
  border-top: 1px solid;
  border-bottom: 1px solid;
  border-color: #007bff;
  border-color: #007bff;
  border-color: #007bff;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with colored text and background 1`] = `
.c0 {
  color: #f0f0f0;
  background-color: #343a40;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex grow 1`] = `
.c0 {
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex grow 2`] = `
.c0 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex shrink 1`] = `
.c0 {
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex shrink 2`] = `
.c0 {
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex wrap 1`] = `
.c0 {
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex wrap 2`] = `
.c0 {
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with flex wrap 3`] = `
.c0 {
  -webkit-flex-wrap: wrap-reverse;
  -ms-flex-wrap: wrap-reverse;
  flex-wrap: wrap-reverse;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with hidden text 1`] = `
.c0 {
  font: 0/0 a;
  color: transparent;
  text-shadow: none;
  background-color: transparent;
  border: 0;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Styles match Div with italic 1`] = `
.c0 {
  font-style: italic;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with margin and size 1`] = `
.c0 {
  margin-left: 0rem;
  margin-right: 0rem;
  margin-top: 0rem;
  margin-bottom: 0rem;
}

@media (min-width:768px) {
  .c0 {
    margin-left: 0.5rem;
    margin-right: 0.5rem;
    margin-top: 0.5rem;
    margin-bottom: 0.5rem;
  }
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with monospace 1`] = `
.c0 {
  font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with order 1`] = `
.c0 {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with padding and margin 1`] = `
.c0 {
  margin-left: 0.25rem;
  margin-right: 0.25rem;
  margin-top: 0.25rem;
  margin-bottom: 0.25rem;
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  padding-top: 0.25rem;
  padding-bottom: 0.25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with resize 1`] = `
.c0 {
  resize: none;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with resize 2`] = `
.c0 {
  resize: both;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with resize 3`] = `
.c0 {
  resize: horizontal;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with resize 4`] = `
.c0 {
  resize: vertical;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with roundedCircle 1`] = `
.c0 {
  border-radius: 50%;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with roundedPill 1`] = `
.c0 {
  border-radius: 50rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with text decoration 1`] = `
.c0 {
  -webkit-text-decoration: none!important;
  text-decoration: none!important;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with text reset 1`] = `
.c0 {
  color: inherit;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with truncated text 1`] = `
.c0 {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Styles match Div with visibilities 1`] = `
.c0 {
  visibility: visible;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with visibilities 2`] = `
.c0 {
  visibility: hidden;
}

<div
  className="c0"
/>
`;

exports[`Styles match Div with word break 1`] = `
.c0 {
  word-break: break-word;
  overflow-wrap: break-word;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Styles match Divs with displays 1`] = `
.c0 {
  display: none;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 2`] = `
.c0 {
  display: inline;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 3`] = `
.c0 {
  display: inline-block;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 4`] = `
.c0 {
  display: block;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 5`] = `
.c0 {
  display: table;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 6`] = `
.c0 {
  display: table-cell;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 7`] = `
.c0 {
  display: table-row;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 8`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with displays 9`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with flex direction 1`] = `
.c0 {
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with flex direction 2`] = `
.c0 {
  -webkit-flex-direction: row-reverse;
  -ms-flex-direction: row-reverse;
  flex-direction: row-reverse;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with flex direction 3`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with flex direction 4`] = `
.c0 {
  -webkit-flex-direction: column-reverse;
  -ms-flex-direction: column-reverse;
  flex-direction: column-reverse;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with float 1`] = `
.c0 {
  float: right;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with float 2`] = `
.c0 {
  float: left;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with float 3`] = `
.c0 {
  float: none;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 1`] = `
.c0 {
  -webkit-box-pack: start;
  -webkit-justify-content: start;
  -ms-flex-pack: start;
  justify-content: start;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 2`] = `
.c0 {
  -webkit-box-pack: end;
  -webkit-justify-content: end;
  -ms-flex-pack: end;
  justify-content: end;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 3`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 4`] = `
.c0 {
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 5`] = `
.c0 {
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with justify content 6`] = `
.c0 {
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with overflow 1`] = `
.c0 {
  overflow: auto;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with overflow 2`] = `
.c0 {
  overflow: hidden;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with paddings 1`] = `
.c0 {
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  padding-top: 0.25rem;
  padding-bottom: 0.25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with paddings 2`] = `
.c0 {
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  padding-top: 0.25rem;
  padding-bottom: 0.25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with paddings 3`] = `
.c0 {
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  padding-top: 0.25rem;
  padding-bottom: 0.25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with position 1`] = `
.c0 {
  position: static;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with position 2`] = `
.c0 {
  position: relative;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with position 3`] = `
.c0 {
  position: absolute;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with position 4`] = `
.c0 {
  position: fixed;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with position 5`] = `
.c0 {
  position: -webkit-sticky;
  position: sticky;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with rounded 1`] = `
.c0 {
  border-top-left-radius: .25rem;
  border-top-right-radius: .25rem;
  border-bottom-left-radius: .25rem;
  border-bottom-right-radius: .25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with rounded 2`] = `
.c0 {
  border-top-left-radius: .25rem;
  border-top-right-radius: .25rem;
  border-bottom-left-radius: .25rem;
  border-bottom-right-radius: .25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with rounded 3`] = `
.c0 {
  border-top-left-radius: .25rem;
  border-top-right-radius: .25rem;
  border-bottom-left-radius: .25rem;
  border-bottom-right-radius: .25rem;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with shadow 1`] = `
.c0 {
  box-shadow: none;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with shadow 2`] = `
.c0 {
  box-shadow: 0 .25rem .5rem rgba(0,0,0,.1);
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with shadow 3`] = `
.c0 {
  box-shadow: 0 .75rem 1.5rem rgba(0,0,0,.2);
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with shadow 4`] = `
.c0 {
  box-shadow: 0 1.25rem 3.5rem rgba(0,0,0,.25);
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text align 1`] = `
.c0 {
  text-align: center;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text align 2`] = `
.c0 {
  text-align: left;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text align 3`] = `
.c0 {
  text-align: right;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text align 4`] = `
.c0 {
  text-align: justify;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text weight 1`] = `
.c0 {
  font-weight: bold;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text weight 2`] = `
.c0 {
  font-weight: bolder;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text weight 3`] = `
.c0 {
  font-weight: normal;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text weight 4`] = `
.c0 {
  font-weight: light;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text weight 5`] = `
.c0 {
  font-weight: lighter;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text wrap 1`] = `
.c0 {
  white-space: normal;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with text wrap 2`] = `
.c0 {
  white-space: nowrap;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with transform 1`] = `
.c0 {
  text-transform: lowercase;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with transform 2`] = `
.c0 {
  text-transform: uppercase;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with transform 3`] = `
.c0 {
  text-transform: capitalize;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with vertical align 1`] = `
.c0 {
  vertical-align: baseline;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with vertical align 2`] = `
.c0 {
  vertical-align: top;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with vertical align 3`] = `
.c0 {
  vertical-align: bottom;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with vertical align 4`] = `
.c0 {
  vertical-align: text-top;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with vertical align 5`] = `
.c0 {
  vertical-align: text-bottom;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with width/height 1`] = `
.c0 {
  width: 100%;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with width/height 2`] = `
.c0 {
  height: 50%;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with width/height 3`] = `
.c0 {
  max-width: 75%;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with width/height 4`] = `
.c0 {
  max-height: 25%;
}

<div
  className="c0"
/>
`;

exports[`Styles match Divs with width/height 5`] = `
.c0 {
  width: auto;
}

<div
  className="c0"
/>
`;

exports[`Styles match hidden Div 1`] = `
.c0 {
  display: none;
}

<div
  className="c0"
  hidden={true}
/>
`;
